### PR TITLE
Add issuer and rsa auth suffixes

### PR DESCRIPTION
**Pull request by tangrammer**

### CODE REVIEW

**Comment by tangrammer** [comment]
now we use auth0 in `test` and `prod`

File: k8s/config/dev.yaml [hunk 1]
```diff
@@ -4,13 +4,15 @@ metadata:
...
+  auth.public.client.id: D5LayiXP1pzq-6g2B_QVvzCw_eycZxQK
+  auth.issuer.suffix.url: /
+  auth.rsa.suffix.url: .well-known/jwks.json
   auth0.public.client.id: D5LayiXP1pzq-6g2B_QVvzCw_eycZxQK
   auth0.url: https://akvotest.eu.auth0.com
   email.host: mail.akvo.org
   environment: test
   flow.api.auth0.root: https://api-auth0.akvotest.org/flow
-  flow.api.root: https://api-auth0.akvotest.org/flow
+  flow.api.root: https://api.akvotest.org/flow
```
Comment (dlebrero) [inline k8s/config/dev.yaml]: why not https://api-auth0.akvotest.org/flow if we are going to use auth0?

Comment (tangrammer) [reply k8s/config/dev.yaml]: good catch!

**Comment by dlebrero** [comment]
Remember that you need to apply the changes manually!

**Pull request merged**